Template/azuredeploy.json

```diff
@@ -86,12 +93,14 @@
         "functionAppName": "[concat(parameters('uniqueSolutionPrefix'), 'function')]",
         "gitUsername": "Azure",
         "functionZipBinary": "https://github.com/Azure/iotedge-lorawan-starterkit/releases/download/v2.0.0/function-2.0.0.zip",
+        "discoveryServiceZipBinary": "TODO:replacewithnextrelease",
```

Comment:
Assuming next release is 2.1.0, you could just use 'https://github.com/Azure/iotedge-lorawan-starterkit/releases/download/v2.1.0/discoveryservice-2.1.0.zip'

Otherwise, if we are not yet sure, let's maybe create a checklist of items that need to be changed for next release (inside a GitHub issue item tracking 2.1.0 release itself)

Reply:
Agreed, I had already created an item for this and linked it in the PR description: #1521. I will change the title to include v2.1.0 to make it easier to find.